pkg/operator/operator.go

```diff
@@ -421,6 +430,18 @@ func (optr *Operator) getCAsFromConfigMap(namespace, name, key string) ([]byte,
 	}
 }
 
+func (optr *Operator) getCloudConfigFromConfigMap(namespace, name, key string) (string, error) {
+	cm, err := optr.kubeClient.CoreV1().ConfigMaps(namespace).Get(name, metav1.GetOptions{})
```

Comment:
We have `optr.clusterCmLister.ConfigMaps(namespace)` now.

Reply:
done